apps/condo/migrations/20220616205429-0137_alter_ticketsource_type.js

```diff
@@ -26,8 +26,8 @@ exports.up = async (knex) => {
 exports.down = async (knex) => {
     await knex.raw(`
         BEGIN;
-            DELETE FROM "MeterReadingSource" WHERE id = '830d1d89-2d17-4c5b-96d1-21b5cd01a6d3';
-            DELETE FROM "MeterReadingSource" WHERE id = '291e093c-050b-4dbd-96d1-da67c5826b6d';
+            DELETE FROM "TicketSource" WHERE id = '830d1d89-2d17-4c5b-96d1-21b5cd01a6d3';
```

Comment:
You have changed old migration

Reply:
There was a mistake here.

Reply:
Adding TicketSource, but removing MeterReadingSource

Reply:
If mistake was mad, then fix it in new migration

Reply:
ok, I revert this change

Reply:
dropped this changes